bin/generate-zbm

```diff
@@ -1052,7 +1059,11 @@ Manually specify the output image prefix; supersedes I<Kernel.Prefix>
 
 =item B<--initcpio|-i>
 
-Use mkinitcpio instead of dracut.
+Force the use of mkinitcpio instead of dracut.
+
+=item B<--no-initcpio|-i>
```

Comment:
Dracut remains the first-class citizen, because it is preferred by default as long as it can be found.

I wasn't even going to include this option, but threw it in so the command-line options can fully cover the alternatives of undefined, true and false for `$runconf{usecpio}`. As implemented, the only purpose of `--no-initcpio` is to force `generate-zbm` to hard fail if `mkinitcpio` is available and `dracut` is not, rather than falling back to `mkinitcpio` in that case. Adding `--dracut` instead of `--no-initcpio` means we add another variable to track and reconcile, adding more logic.

Eventually, if we add support for other generators, the `InitCPIO` option in the config should go away in favor of a string like

```yaml
Generator: dracut | mkinitcpio | initramfs-tools | ...
```

and the command-line options should mirror those.

For now, I guess I'd rather just drop `--no-initcpio`; what do you think?

Reply:
I'll need to re-read the flow of `generate-zbm`, I clearly misunderstood what `--no-initcpio` was actually doing. I'm fine with `--no-initcpio` being added given your explanation for it's usage.